animalfacts.py

```diff
@@ -2938,6 +2939,24 @@ def animalfactsbot(reddit):
     'The blue whale is the largest animal to have ever lived on Earth; the largest on record was 100 ft. long.'
     )
 
+WILDEBEEST_FACTS = ( 
+    'Wildebeest is one of the largest antelopes. It can reach 8 feet in length, 4.5 feet in height and weight up to 600 pounds.',
```

Comment:
Spelling/grammar check: "are", not "is". "They" instead of "it", and "weigh" instead of "weight".